File: J2465/JS/Aug/21-08/Task FA3.70/scripts/employee.js
const _name = Symbol("name");
const _age = Symbol("age");
const _role = Symbol("role");

class Employee {
  constructor(name, age, role) {
    this[_name] = name;
    this[_age] = age;
    this[_role] = role;
  }

  getInfo() {
    return `Name: ${this[_name]}, Age: ${this[_age]}, Role: ${this[_role]}`;
  }
}

export { Employee };

File: J2465/JAVA/Nov/21-11/Task STR8.10/demo/src/main/java/com/example/demo/model/Employee.java
package com.example.demo.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Table;

@Entity
@Table(name = "employees")
public class Employee {
    public String name;
    public Integer age;
    public String department;
    public Double salary;

    public Employee(String name, Integer age, String department, Double salary) {
        this.name = name;
        this.age = age;
        this.department = department;
        this.salary = salary;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    public String getDepartment() {
        return department;
    }

    public void setDepartment(String department) {
        this.department = department;
    }

    public Double getSalary() {
        return salary;
    }

    public void setSalary(Double salary) {
        this.salary = salary;
    }

    @Override
    public String toString() {
        return "Employee [name=" + name + ", age=" + age + ", department=" + department + ", salary=" + salary + "]";
    }

}
